bob/src/api/mod.rs

```diff
@@ -145,12 +145,25 @@ pub(crate) struct NodeConfiguration {
 }
 
 #[derive(Debug, Serialize)]
-pub(crate) struct SpaceInfo {
+struct Space {
     total_disk_space_bytes: u64,
     free_disk_space_bytes: u64,
     used_disk_space_bytes: u64,
+}
+
+#[derive(Debug, Serialize)]
+pub(crate) struct SpaceInfo {
+    #[serde(flatten)]
+    total_space: Space,
+
+    /// Total occupied disk space by Bob
     occupied_disk_space_bytes: u64,
-    occupied_disk_space_by_disk: HashMap<String, u64>,
+
+    /// Key - Bob disk name
+    occupied_disk_space_by_disk: HashMap<DiskName, u64>,
+    
+    /// Key - Mount point or Bob disk name if no mount point was found
+    disk_space_by_disk: HashMap<String, Space>,
```

Comment:
I just thought that it would be better to use disk name as the key (on Linux it is `sda1`, `sdb1`, on Windows it is `C:`, `D:`), because this is the common way of identification of disks in OS

Reply:
I think, it would be better to do later in another request. Right now, mount point would be enough here